components/script/textinput.rs

```diff
@@ -1164,6 +1166,85 @@ impl<T: ClipboardProvider> TextInput<T> {
     }
 }
 
+pub(crate) fn get_content_from_clipboard_event(event: &ClipboardEvent) -> String {
```

Comment:
Right now this function and `get_content_from_key_event` are only used for `beforeinput`. Is it possible to unify it with the actual text input handling?